src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs

```diff
@@ -90,7 +90,7 @@ protected AbstractGenerateMemberService()
 
             var syntaxFacts = document.Project.LanguageServices.GetService<ISyntaxFactsService>();
             var semanticModel = document.SemanticModel;
-            var isMemberAccessExpression = syntaxFacts.IsMemberAccessExpression(simpleNameOrMemberAccessExpression);
+            var isMemberAccessExpression = syntaxFacts.IsSimpleMemberAccessExpression(simpleNameOrMemberAccessExpression);
```

Comment:
>IsSimpleMemberAccessExpression [](start = 55, length = 30)

Why this name change? Was it always checking if it was a simple member access expression and the method was just poorly named?